src/envoy/alts/tsi_transport_socket.h

```diff
@@ -99,13 +106,15 @@ class TsiSocket : public Network::TransportSocket,
  */
 class TsiSocketFactory : public Network::TransportSocketFactory {
  public:
-  explicit TsiSocketFactory(HandshakerFactory handshaker_factory);
+  explicit TsiSocketFactory(HandshakerFactory handshaker_factory,
+                            HandshakeValidator handshake_validator);
```

Comment:
I think explicit is not needed anymore, as there are more than one argument.

Reply:
Done.